lib/covalence/core/cli_wrappers/terraform_cli.rb

```diff
@@ -34,24 +34,68 @@ def self.terraform_clean(path, dry_run: false, verbose: true)
     end
 
     def self.terraform_check_style(path)
-      output, status = Open3.capture2e(ENV, Covalence::TERRAFORM_CMD, "fmt", "-write=false", path)
-      return false unless status.success?
-      output = output.split("\n")
-      (output.size == 0)
+      cmd = [Covalence::TERRAFORM_CMD, "fmt", "-check"]
+
+      output = PopenWrapper.run(
+          cmd,
+          path,
+          '',
+          ignore_exitcode: false)
+      (output == 0)
     end
 
-    def self.terraform_init(path='', args: '', ignore_exitcode: false)
+    def self.terraform_init(path: '', workdir: Dir.pwd, args: '', ignore_exitcode: false)
       if ENV['TF_PLUGIN_LOCAL'] == 'true'
-        cmd = [Covalence::TERRAFORM_CMD, "init", "-get=false", "-input=false", "-plugin-dir=#{Covalence::TERRAFORM_PLUGIN_CACHE}"]
+        cmd = [Covalence::TERRAFORM_CMD, "init", "-get-plugins=false", "-get=false", "-input=false", "-plugin-dir=#{Covalence::TERRAFORM_PLUGIN_CACHE}"]
```

Comment:
Q: Why was `-get-plugins=false` added here?

Reply:
To make sure we are not downloading any latest version of the plugin since we are using the custom build plugins that are in the container already. During testing, I did run into problem of terraform downloaded the latest version of the plugin and it's not compatible with terraform 0.12.

Reply:
I believe that only happens with a module that specifies a requirement of that.